docs/github-labels.md

```diff
@@ -0,0 +1,108 @@
+# GitHub Label List
+
+We use GitHub labels to perform issue triage, track and report on development
+progress, plan roadmaps, and automate issue grooming.
+
+To ensure that contributing new issues and PRs remains straight-forward, we would
+like to keep the labels required for submission to a minimum. The remaining
+labels will be added either by automation or manual grooming by other
+contributors and maintainers.
+
+This labels in this list originated within Kubernetes at
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md.
+
+## Labels that apply to issues or PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| api-review                         | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                           | Issues or PRs related to an API | Any |
+| area/blog                          | Issues or PRs related to blog entries. | Any |
+| area/dependency                    | Issues or PRs related to dependency changes | Any |
+| area/endpoint/identity             | Issues or PRs related to endpoint identity | Any |
```

Comment:
are the endpoint* labels relevant?

Reply:
I'm using the `endpoint` as a generic way to refer to a workload endpoint (whether it be a container or VM). NSX uses `port`. I had intended these labels to categorize work as follows:
* `area/endpoint/identity` -- how we attach metadata to an endpoint for matching traffic or selecting it has the target of a policy. Metadata could be Kubernetes labels, other Kubernetes metadata such as service account, policy precedent group when we create "tiers", etc. Also could relate to work on planned Northbound federation API in regards to how we import remote inventory for policy evaluation.
* `area/endpoint/selection` -- how endpoints are selected for traffic match or policy application. 
* `area/endpoint/type` -- type type of endpoint (container or VM). (We currently aren't supporting VMs with Antrea; however, VMs may be targets in a federated context.

This may be too detailed and maybe we couch all of this under `area/endpoint`. Open to suggestions.

It's fair that we should add some additional information describing what we intend to categorize with the label if the meaning is not clear from the name alone.